cmd/object-handlers.go

```diff
@@ -60,7 +60,7 @@ var supportedHeadGetReqParams = map[string]string{
 }
 
 const (
-	compressionAlgorithmV1 = "golang/snappy/LZ77"
+	compressionAlgorithmV1 = "klauspost/compress/LZ77"
```

Comment:
```suggestion
	compressionAlgorithmV1 = "klauspost/compress/s2"
```

Not sure if this should be v2?